rbac/management/access/view.py

```diff
@@ -78,21 +86,31 @@ class AccessView(APIView):
     pagination_class = api_settings.DEFAULT_PAGINATION_CLASS
     permission_classes = (AllowAny,)
 
-    def get_queryset(self):
+    def get_queryset(self, ordering):
         """Define the query set."""
-        return get_access_queryset(self.request)
+        access_queryset = get_access_queryset(self.request)
+        if ordering != "unordered":
+            if ordering[0] == "-":
+                order_sign = "-"
+                field = ordering[1:]
+            else:
+                order_sign = ""
+                field = ordering
+            return access_queryset.order_by(f"{order_sign}permission__{field}")
+        return access_queryset
 
     def get(self, request):
         """Provide access data for principal."""
-        validate_limit_and_offset(request.query_params)
-        app = request.query_params.get(APPLICATION_KEY)
+        # Parameter extraction
+        sub_key, ordering = self.validate_and_get_param(request.query_params)
+
         principal = get_principal_from_request(request)
         cache = AccessCache(request.tenant.schema_name)
-        access_policy = cache.get_policy(principal.uuid, app)
+        access_policy = cache.get_policy(principal.uuid, sub_key)
         if access_policy is None:
-            queryset = self.get_queryset()
+            queryset = self.get_queryset(ordering)
```

Comment:
Just so I'm understanding (tested locally and it seems to work), if I send a request with `?application=rbac` for an initial request, my `sub_key` value would be `rbac`, and thus cached with that key (as it would be today), but a subsequent request with `order_by=verb` would cause my key to change, and would cache based on the ordering, correct?

Reply:
Yep, that's right